prometheus_client/core.py

```diff
@@ -329,6 +347,19 @@ def _samples(self):
 
 @_MetricWrapper
 class Summary(object):
+    '''
+    A Summary tracks the size and number of events.
+
+    Example use cases for Summaries:
+    - Response latency
+    - Request size
+
+    Example for a Summary:
+
+    from prometheus_client import Summary
+    s = Summary('request_latency_seconds', 'Description of summary')
+    s.observe(4.7)    # Observe 4.7 (seconds in this case)
```

Comment:
I agree, that could be misleading. Since that example is from the README file, should I go ahead and change it there as well?